pom.xml

```diff
@@ -272,6 +272,13 @@
                 </container>
               </configuration>
             </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-javadoc-plugin</artifactId>
+              <configuration>
+                <source>${java-version}</source>
+              </configuration>
+            </plugin>
```

Comment:
- [ ] Is it enough to use `source` property?

Reply:
I think it's enough to use `source` property.
I have confirmed that javadoc can be generated below:
- java 1.8.0_172-b11
- java 11+28
- java 11.0.5+10-LTS

I think that other properties set by gfw should be set originally, and `source` property should be set by the user consciously.